memory/src/main/scala/filodb.memory/format/vectors/HistogramCompressor.scala

```diff
@@ -2,85 +2,214 @@ package filodb.memory.format.vectors
 
 import scala.io.Source
 
-import org.agrona.ExpandableArrayBuffer
+import org.agrona.{ExpandableArrayBuffer, ExpandableDirectByteBuffer}
+import org.agrona.concurrent.UnsafeBuffer
+import scalaxy.loops._
 
 import filodb.memory.NativeMemoryManager
 import filodb.memory.format._
 
 //scalastyle:off
+trait CompressorAnalyzer {
```

Comment:
Is this just for bench marking? Why is it here?

Reply:
I had to reuse the code in there for multiple Apps.  One for size-based stuff, one for the time-based standalone app you requested for standalone benchmarking.  One for Prom-style data model sizing.